### PR TITLE
Pr for testing

### PR DESCRIPTION
**Pull request by kittinunf**
Test PR, _NOT MEANT TO BE MERGED_

### CODE REVIEW

File: .idea/vcs.xml [hunk 1]
```diff
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
   <component name="VcsDirectoryMappings">
-    <mapping directory="" vcs="Git" />
     <mapping directory="$PROJECT_DIR$" vcs="Git" />
   </component>
-</project>
\ No newline at end of file
+</project>
```
Comment (kittinunf) [inline .idea/vcs.xml]: Hmm, I am not sure about this?

Comment (kindraywind) [reply .idea/vcs.xml]: Why?

File: app/src/main/kotlin/com/octtoplus/proj/recorda/MainActivity.kt [hunk 2]
```diff
@@ -10,7 +10,8 @@ import android.widget.Toast
 import kotlinx.android.synthetic.main.activity_main.*
 import kotlinx.android.synthetic.main.content_main.*
 
-class MainActivity : AppCompatActivity() {
+class MainActivity : AppCompatActivity() { aa
```
Comment (kittinunf) [inline app/src/main/kotlin/com/octtoplus/proj/recorda/MainActivity.kt:13]: 1. This is not complied?

File: app/src/main/kotlin/com/octtoplus/proj/recorda/MainActivity.kt [hunk 3]
```diff
@@ -10,7 +10,8 @@ import android.widget.Toast
...
+
```
Comment (kittinunf) [inline app/src/main/kotlin/com/octtoplus/proj/recorda/MainActivity.kt:14]: Add empty line?

File: .idea/encodings.xml [hunk 4]
```diff
@@ -1,6 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (kittinunf) [inline .idea/encodings.xml:1]: This will change a lot in our PR.

File: .idea/encodings.xml [hunk 5]
```diff
@@ -1,6 +0,0 @@
...
-<project version="4">
-  <component name="Encoding">
-    <file url="PROJECT" charset="UTF-8" />
```
Comment (kittinunf) [inline .idea/encodings.xml:4]: This too?

File: .idea/vcs.xml [hunk 6]
```diff
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
```
Comment (kittinunf) [inline .idea/vcs.xml:1]: Hello?

File: .idea/vcs.xml [hunk 7]
```diff
@@ -1,7 +1,6 @@
...
 <project version="4">
```
Comment (kittinunf) [inline .idea/vcs.xml:2]: Hello 2

File: .idea/vcs.xml [hunk 8]
```diff
@@ -1,7 +1,6 @@
...
   <component name="VcsDirectoryMappings">
-    <mapping directory="" vcs="Git" />
     <mapping directory="$PROJECT_DIR$" vcs="Git" />
```
Comment (kittinunf) [inline .idea/vcs.xml:4]: Hello 5

File: .idea/vcs.xml [hunk 9]
```diff
@@ -1,7 +1,6 @@
...
   </component>
```
Comment (kittinunf) [inline .idea/vcs.xml:6]: Hello 5 >

File: .idea/vcs.xml (hunk 8 above)
Comment (kittinunf) [inline .idea/vcs.xml:4]: Hello 4 >

File: .idea/gradle.xml [hunk 10]
```diff
@@ -1,19 +0,0 @@
...
-          <set>
-            <option value="$PROJECT_DIR$" />
-            <option value="$PROJECT_DIR$/app" />
-          </set>
-        </option>
-        <option name="resolveModulePerSourceSet" value="false" />
-      </GradleProjectSettings>
-    </option>
-  </component>
-</project>
```
Comment (kittinunf) [inline .idea/gradle.xml:19]: Hello 19 <

File: .idea/gradle.xml [hunk 11]
```diff
@@ -3,15 +3,8 @@
...
+        <option name="distributionType" value="DEFAULT_WRAPPED" />
         <option name="externalProjectPath" value="$PROJECT_DIR$" />
-        <option name="gradleHome" value="$APPLICATION_HOME_DIR$/gradle/gradle-2.14.1" />
-        <option name="modules">
-          <set>
-            <option value="$PROJECT_DIR$" />
-            <option value="$PROJECT_DIR$/app" />
-          </set>
-        </option>
         <option name="resolveModulePerSourceSet" value="false" />
```
Comment (kittinunf) [inline .idea/gradle.xml:8]: Hello 8 >

File: .idea/gradle.xml [hunk 12]
```diff
@@ -3,15 +3,8 @@
   <component name="GradleSettings">
     <option name="linkedExternalProjectsSettings">
       <GradleProjectSettings>
-        <option name="distributionType" value="LOCAL" />
+        <option name="distributionType" value="DEFAULT_WRAPPED" />
         <option name="externalProjectPath" value="$PROJECT_DIR$" />
```
Comment (kittinunf) [inline .idea/gradle.xml:7]: Hello 7 >

File: app/src/main/kotlin/com/octtoplus/proj/recorda/ThirdActivity.kt [hunk 13]
```diff
@@ -9,4 +9,10 @@ class ThirdActivity : AppCompatActivity() {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_third)
     }
+
+    1
+    2
+
+    3
```
Comment (kittinunf) [inline app/src/main/kotlin/com/octtoplus/proj/recorda/ThirdActivity.kt:16]: Hello 16 >

File: app/src/main/kotlin/com/octtoplus/proj/recorda/ThirdActivity.kt [hunk 14]
```diff
@@ -9,4 +9,10 @@ class ThirdActivity : AppCompatActivity() {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_third)
     }
+
```
Comment (kittinunf) [inline app/src/main/kotlin/com/octtoplus/proj/recorda/ThirdActivity.kt:12]: Hello 12 >

File: app/src/main/kotlin/com/octtoplus/proj/recorda/MainActivity.kt [hunk 15]
```diff
@@ -54,4 +55,4 @@ class MainActivity : AppCompatActivity() {
         return super.onOptionsItemSelected(item)
     }
 
-}
\ No newline at end of file
+}
```
Comment (kittinunf) [inline app/src/main/kotlin/com/octtoplus/proj/recorda/MainActivity.kt:58]: Hello 58 >

File: .idea/gradle.xml [hunk 16]
```diff
@@ -3,15 +3,8 @@
...
-        <option name="gradleHome" value="$APPLICATION_HOME_DIR$/gradle/gradle-2.14.1" />
```
Comment (kittinunf) [inline .idea/gradle.xml:8]: Hello 8 <

File: .idea/gradle.xml (hunk 11 above)
Comment (kittinunf) [inline .idea/gradle.xml:8]: Hello 8 >

File: .idea/gradle.xml (hunk 12 above)
Comment (kittinunf) [inline .idea/gradle.xml:7]: Hello 7 >

**Pull request closed without merging**